lib/elasticsearch/config.js

```diff
@@ -99,7 +99,8 @@ const FILTER_CONFIG = {
   dateBefore: {
     operator: 'custom',
     type: 'int'
-  }
+  },
+  genre: { operator: 'match', field: ['genreForm.raw'], repeatable: true }
```

Comment:
Sorry to notice this late, but please change this to `genreForm` to match the property name.